packages/wonder-blocks-icon/generated-snapshot.test.js

```diff
@@ -0,0 +1,143 @@
+// @flow
+// This file is auto-generated by gen-snapshot-tests.js
+// Do not edit this file.  To make changes to these snapshot tests:
+//   1. edit the markdown documentation files in the package,
+//        packages/wonder-blocks-icon
+//   2. Run `yarn run gen-snapshot-tests`.
+import React from "react";
+import renderer from "react-test-renderer";
+import Icon from "./components/icon.js";
+
+describe("wonder-blocks-icon", () => {
+    it("example 1", () => {
+        const {
+            default: Icon,
+            icons,
+        } = require("@khanacademy/wonder-blocks-icon");
+
+        const example = <Icon icon={icons.search} size="small" />;
+        const tree = renderer.create(example).toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+    it("example 2", () => {
+        const {StyleSheet, css} = require("aphrodite");
+        const {View} = require("@khanacademy/wonder-blocks-core");
+        const {
+            default: Icon,
+            icons,
+        } = require("@khanacademy/wonder-blocks-icon");
+
+        const headings = ["small", "medium"];
+
+        const styles = StyleSheet.create({
+            emptyCell: {
+                background: "#EEE",
+            },
+            nameCell: {
+                fontFamily: "monospace",
+                textAlign: "right",
+                paddingRight: 4,
+            },
+            iconCell: {
+                minWidth: 75,
+                textAlign: "center",
+                padding: 4,
+            },
+            tableBorder: {
+                border: "1px #DDD solid",
+            },
+        });
+
+        const example = (
+            <table
+                className={css(styles.table)}
+                style={{borderCollapse: "collapse"}}
+            >
```

Comment:
This table makes it really clear which icons have only one size or the other.  Thanks.